### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by nikolas**
Not everything here is perfect, but hey it's a start.

### CODE REVIEW

File: app/locales/es/translations.js [hunk 1]
```diff
@@ -1,11 +1,160 @@
...
-  //
-  // "a": {
-  //   "nested": {
-  //     "key": "Text for a.nested.key"
-  //   }
-  // },
-  //
-  // "key.with.interpolation": "Text with {{anInterpolation}}"
+  'site_title': 'datafruits',
+  'site_title_fm': 'datafruits.fm',
```
Comment (sharpshark28) [inline app/locales/es/translations.js]: No need to specify text that doesn't change for Spanish, it'll default to the English file when a translation is not found. The site title at least can be left out :)

File: app/locales/es/translations.js [hunk 2]
```diff
@@ -1,11 +1,160 @@
...
+    'upcoming_shows': 'PRÓXIMOS PROGRAMAS',
+    'latest_podcasts': 'ÚLTIMOS PÓDCASTS',
+    'events': 'EVENTOS',
+    'podcasts': {
+      'title': 'PODCASTS',
+      'subscribe_request': 'Suscríbase al pódcast datafruits.fm',
+      'link_itunes': 'iTunes',
+      'link_overcast': 'Overcast',
+      'link_podcastsapp': 'Podcasts.app',
+      'link_rss': 'RSS',
```
Comment (sharpshark28) [inline app/locales/es/translations.js]: Ditto for these as they are app names and don't change in Spanish.

**Comment by nikolas** [comment]
alright, I've removed the stuff that doesn't change, like you pointed out.

**Comment by mcfiredrill** [comment]
Looks amazing! Thank you so much!

**Comment by mcfiredrill** [comment]
Build passed, but our builds are still exiting with status 1 for some reason, which makes them get marked as failed. :\

So I'll merge this! Thanks again.

**Pull request merged**